views/chitter_homepage.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Chitter Homepage</h1>
+<% @peeps.each do |peep| %>
+  <ul>
+    <li><%= @user_class.get(peep.user_id).username %></li>
+    <li><%= @user_class.get(peep.user_id).name %></li>
```

Comment:
As per comment above, avoid making database class or including complex logic in your views